### ISSUE TITLE
Learn more heading is not associated with the respective field: A11y_Maow_Home Page_Info and Relationships.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#114]
### GitHub Tags
#A11yTCS; #A11ySev3; #ChromiumEdge; #WCAG1.3.1; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #Info and Relationships; #AILimited: #A11yMAS;

### Actual:
Learn more heading is not associated with the respective field.
**Same issue is observed with follow and community heading.**
### 
### Expected:
Only one H1 heading should be defined on the page.

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till learn more.
4. Verify that the Learn more heading is associated with the respective field or not.

### User Impact:
Screen reader users will not be able to get under which section the controls are present and they are navigating. 

### Attachment:
![Bug114](https://github.com/microsoft/moaw/assets/124244478/35ee3858-10f1-4ad3-a409-4cc5bbdc7db6)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
> @AnKushSingh05 this one has been fixed, could you validate it?

Verified the issue in below environment
https://moaw.dev/
Issue is not fixed. Still we can see that NVDA is not announcing learn more section heading
Please reactivate the bug
![Screenshot (1955)](https://github.com/microsoft/moaw/assets/124244478/caf77f0f-a6b9-42c1-8f17-66d173620b4e)

**Comment by sinedied** [comment]
Thank you, this time I added the missing landmarks and tested it with NVDA, should be good now.
@AnKushSingh05 could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug.
![Screenshot (1973)](https://github.com/microsoft/moaw/assets/124244478/306345f4-1cec-489c-9f5d-27f2fefc7c84)
